cfme/tests/configure/test_rbac_landing_page.py

```diff
@@ -0,0 +1,214 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.base.credential import Credential
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+COMMON_FEATURES = {
```

Comment:
Thoughts on moving these into the data directory?

Reply:
Why do you suggest that?

Reply:
Just a suggestion, so that we separate data/config from tests. I'd say it's optional :+1: